packages/open-payments/src/client/wallet-address.ts

```diff
@@ -1,39 +1,54 @@
 import { HttpMethod } from '@interledger/openapi'
 import { RouteDeps, UnauthenticatedResourceRequestArgs } from '.'
-import { JWKS, WalletAddress, getRSPath } from '../types'
+import { JWKS, WalletAddress, DIDDocument, getWAPath } from '../types'
 import { get } from './requests'
 
 export interface WalletAddressRoutes {
   get(args: UnauthenticatedResourceRequestArgs): Promise<WalletAddress>
   getKeys(args: UnauthenticatedResourceRequestArgs): Promise<JWKS>
+  // TODO: Define schema for DID Document
+  getDIDDocument(args: UnauthenticatedResourceRequestArgs): Promise<DIDDocument>
 }
 
 export const createWalletAddressRoutes = (
   deps: RouteDeps
 ): WalletAddressRoutes => {
-  const { openApi, ...baseDeps } = deps
+  const { axiosInstance, openApi, logger } = deps
```

Comment:
to prevent the type/build errors, we can just pass in the `baseDeps` through ⬇️ 
```suggestion
  const { openApi, ...baseDeps } = deps
```